tests/pruners_tests/test_hyperband.py

```diff
@@ -19,15 +20,25 @@ def test_hyperband_experimental_warning() -> None:
 
     with pytest.warns(optuna.exceptions.ExperimentalWarning):
         optuna.pruners.HyperbandPruner(
-            min_resource=MIN_RESOURCE, reduction_factor=REDUCTION_FACTOR, n_brackets=N_BRACKETS
+            max_resource=MAX_RESOURCE, min_resource=MIN_RESOURCE, reduction_factor=REDUCTION_FACTOR
+        )
+
+
+def test_hyperband_deprecation_warning() -> None:
+    with pytest.deprecated_call():
+        optuna.pruners.HyperbandPruner(
+            max_resource=MAX_RESOURCE,
+            min_resource=MIN_RESOURCE,
```

Comment:
A nit but please reorder the arguments as mentioned in https://github.com/optuna/optuna/pull/1138/files#r411226640. 
```suggestion
            min_resource=MIN_RESOURCE,
            max_resource=MAX_RESOURCE,
```